### PR TITLE
💪🏿Chore: Import article and menu less files

### PR DESCRIPTION
**Pull request by ogwurujohnson**

### CODE REVIEW

File: Article/Article.less [hunk 1]
```diff
@@ -39,6 +39,22 @@
...
+    color: white;
+    cursor: pointer;
+    transform: translate(-50%);
+    background-color: white;
+    background-color: red;
+    width: 2%;
+    
+    text-align: center;
+    border: 1px solid black;
+  }
```
Comment (mattshardman) [inline Article/Article.less]: I like to use these for buttons: 
https://material.io/tools/icons/?style=baseline

**Comment by mattshardman** [comment]
Great job Johnson!

- You have correctly implemented the menu and articles components
- You have added some of your own articles
- You git work flow is amazing
- You have also gone above the MVP requirements and added GSAP animations and the delete functionality

**Possible things to work on**
- Not a lot to say, you could try adding an expand animation for the articles

Overall great job, you have implemented everything correctly and clearly understand the concept of using components well. Also good job having a play around with different features of GSAP. Keep up the good work!

File: CSS/index.css [hunk 2]
```diff
@@ -35,6 +35,90 @@
...
+  transform: translate(-50%);
+  background-color: white;
+  background-color: red;
+  width: 2%;
+  text-align: center;
+  border: 1px solid black;
+}
+.article-open {
+  height: 400px;
+}
```
Comment (mattshardman) [inline CSS/index.css]: i would probably not use absolute pixels here for dynamic content

Comment (ogwurujohnson) [reply CSS/index.css]: ok, would change that

File: Article/Article.less (hunk 1 above)
Comment (ogwurujohnson) [reply Article/Article.less]: would check it out

Comment (ogwurujohnson) [reply Article/Article.less]: Thanks, it would really come in handy

**Pull request merged**